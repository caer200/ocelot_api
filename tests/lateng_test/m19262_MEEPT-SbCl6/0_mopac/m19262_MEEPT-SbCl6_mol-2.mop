CHARGES

 
Sb	3.445963 +1 3.881901 +1 6.490933 +1
Cl	3.328687 +1 1.541265 +1 6.245903 +1
Cl	3.582758 +1 6.247340 +1 6.756012 +1
Cl	5.817873 +1 3.721476 +1 6.481662 +1
Cl	1.094837 +1 4.016737 +1 6.480470 +1
Cl	3.486662 +1 4.156691 +1 4.145956 +1
Cl	3.437981 +1 3.682501 +1 8.841829 +1
