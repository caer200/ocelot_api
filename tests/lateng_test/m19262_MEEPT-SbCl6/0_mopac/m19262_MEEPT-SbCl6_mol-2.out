 *******************************************************************************
 ** Site#: 29200        For non-commercial use only    Version 19.090L 64BITS **
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 19.090L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 175**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Mon Oct  7 22:31:20 2019  *
 *  CHARGES    - IDENTIFY AND PRINT CHARGED ATOMS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *******************************************************************************
CHARGES


   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Sb          3.44596300  *   3.88190100  *   6.49093300  *
     2      Cl          3.32868700  *   1.54126500  *   6.24590300  *
     3      Cl          3.58275800  *   6.24734000  *   6.75601200  *
     4      Cl          5.81787300  *   3.72147600  *   6.48166200  *
     5      Cl          1.09483700  *   4.01673700  *   6.48047000  *
     6      Cl          3.48666200  *   4.15669100  *   4.14595600  *
     7      Cl          3.43798100  *   3.68250100  *   8.84182900  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Sb          3.4460      3.8819      6.4909
     2        Cl          3.3287      1.5413      6.2459
     3        Cl          3.5828      6.2473      6.7560
     4        Cl          5.8179      3.7215      6.4817
     5        Cl          1.0948      4.0167      6.4805
     6        Cl          3.4867      4.1567      4.1460
     7        Cl          3.4380      3.6825      8.8418

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 http://www.springerlink.com/openurl.asp?genre=article&id=doi:10.1007/s00894-012-1667-x

           Empirical Formula: Sb Cl6  =     7 atoms



      MOLECULAR POINT GROUP   :   Ci  

   TOPOGRAPHY OF SYSTEM

   ATOM No.    LABEL   Atoms connected to this atom
      1         Sb      2      3      4      5      6      7
      2         Cl      1
      3         Cl      1
      4         Cl      1
      5         Cl      1
      6         Cl      1
      7         Cl      1

     Ion Atom No.  Type    Charge
 
    1       1     Sb       -1

          COMPUTED CHARGE ON SYSTEM: -1

          COMPUTED CHARGE ON SYSTEM:  -1

          JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED

 ******************************************************************************
 *                                                                            *
 *     Error and normal termination messages reported in this calculation     *
 *                                                                            *
 * JOB STOPPED HERE BECAUSE KEYWORD "CHARGES" WAS USED                        *
 * JOB ENDED NORMALLY                                                         *
 *                                                                            *
 ******************************************************************************



 TOTAL JOB TIME:             0.00 SECONDS

 == MOPAC DONE ==
